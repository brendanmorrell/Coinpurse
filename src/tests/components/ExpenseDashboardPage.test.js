import React from 'react';
import { shallow } from 'enzyme';
// no idea why but had to install "css-loader" via yarn and then add:
// {
//   "snapshotSerializers": ["enzyme-to-json/serializer"],
//   "moduleNameMapper": {"^.+\\.css$":"css-loader"},
// }
// into the jest config file to get this to work. Found this answer in the Q&A. Seems to allow jest to read the css from date-picker
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';


test('should render the ExpenseListFilters and ExpenseList components', () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
