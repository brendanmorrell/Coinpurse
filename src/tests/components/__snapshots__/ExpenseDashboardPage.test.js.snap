// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the ExpenseListFilters and ExpenseList components 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
